package.json

```diff
@@ -18,21 +18,37 @@
     "@angular/platform-browser": "^16.2.0",
     "@angular/platform-browser-dynamic": "^16.2.0",
     "@angular/router": "^16.2.0",
+    "@ngrx/effects": "^16.3.0",
+    "@ngrx/store": "^16.3.0",
+    "axios": "^1.6.2",
+    "cheerio": "^1.0.0-rc.12",
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
+    "luxon": "3.4.4",
+    "ngrx-store-localstorage": "^16.1.0",
     "rxjs": "~7.8.0",
+    "tailwind": "^4.0.0",
```

Comment:
i like this very much ;-) 
btw. works also with ndbx (e.g. replacement for ndbx flex utils)